scss/detail.scss

```diff
@@ -172,3 +172,72 @@
   line-height: 21px;
   margin: 25px 250px 25px 25px;
 }
+
+.login-inputs {
+  align-items: center;
+  background-color: $wild-sand;
+  border-top: 5px solid $cerulean;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: center;
+  margin-top: 75px;
+  width: 300px;
+}
+
+.login-container {
+  display: flex;
+  justify-content: center;
+}
+
+.form-button-login {
+  background-color: $earls-green;
+  border-radius: 5px;
+  color: $white;
+  height: 32px;
+  margin-bottom: 35px;
+  width: 252px;
+
+  &::before {
+    background: $silver-chalice;
+    content: "";
+    display: flex;
+    height: 2px;
+    position: relative;
+    top: 40px;
+    width: 252px;
+  }
+}
+
+.form-button-singup {
+  background-color: $wild-sand;
+  border: 2px solid $earls-green;
+  border-radius: 5px;
+  box-sizing: border-box;
+  color: $earls-green;
+  height: 32px;
+  width: 252px;
+}
+
+.form-container {
+  display: flex;
+  justify-content: center;
+}
+
+.form-login {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-input {
+  background-color: $white;
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
```

Comment:
you use a lot this value use on principal container and for other use with relative to this.